feat: openapi typescript client (#2885)
Redocly/redocly-cli: .changeset/openapi-typescript.md

---
'@redocly/openapi-typescript': minor
'@redocly/cli': minor
---

> Add the `generate-client` command (**experimental**): generate a typed, zero-dependency TypeScript client from an OpenAPI description.

Reviewer: ```suggestion
Added an **experimental** `generate-client` command that generates a typed, zero-dependency TypeScript client from an OpenAPI description.
```
